apps.yml

```diff
@@ -4028,6 +4028,7 @@ apps:
     authorized_groups:
     - team_secops
     - team_opsec
+    - team_netops
```

Comment:
I presume secops is okay with this extension of access to a formerly-security-only group?